exchanges/bybit/bybit_websocket.go

```diff
@@ -379,24 +382,44 @@ func (by *Bybit) wsHandleData(respRaw []byte) error {
 
 			switch e {
 			case wsAccountInfo:
-				var data []wsAccount
+				var data []WsAccount
 				err := json.Unmarshal(respRaw, &data)
 				if err != nil {
 					return fmt.Errorf("%v - Could not convert to outboundAccountInfo structure %w",
 						by.Name,
 						err)
 				}
-				by.Websocket.DataHandler <- data
+
+				if len(data) == 0 {
+					return fmt.Errorf("%v - no account data returned", by.Name)
+				}
+
+				balanceChanges := make([]account.Change, len(data[0].Balance))
+				for x := range data[0].Balance {
+					balanceChanges[x] = account.Change{
+						Exchange: by.Name,
+						Asset:    asset.Spot,
+						Currency: currency.NewCode(data[0].Balance[x].Asset),
+						Amount:   data[0].Balance[x].Available.Float64(),
+					}
+				}
+
+				by.Websocket.DataHandler <- balanceChanges
 				return nil
 			case wsOrderExecution:
-				var data []wsOrderUpdate
+				var data []WsOrderUpdate
 				err := json.Unmarshal(respRaw, &data)
 				if err != nil {
 					return fmt.Errorf("%v - Could not convert to executionReport structure %w",
 						by.Name,
 						err)
 				}
 
+				avail, err := by.GetAvailablePairs(asset.Spot)
```

Comment:
Why not enabled?

Reply:
<img width="504" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/d0a75dcd-5480-4209-a516-1abde199b33e">

Reply:
I was hoping for a more detailed explanation because I don't believe it's a good addition. I always read PR descriptions and hassle you when you don't populate them

Reply:
Private websocket push data should not be restricted by a wrapper in any case. It enables key sharing and consolidation of trading bots while maintaining comprehensive risk monitoring. Unrestricted access ensures flexibility.

Reply:
As trades/tickers and orderbook subscriptions are currently based on enabled pairs, we should keep this as is and in a future update add in flexibility which allows the user to choose between all updates for all available pairs or only filtered ones (their enabled pairs)

Reply:
I will change but I am coming to the opinion however that whatever is returned by the websocket reader, I would advocate to never filter data on this layer because this can be managed else ware by clients/ GCT's engine layer especially account level data. In context when we fail to process order side/type we still try to process that order.

Reply:
Its a good opinion, but we'll need to have those changes in place first. Enabled/Available pairs is something that can be managed by clients right now at least